app/src/main/java/co/touchlab/droidconandroid/shared/presenter/UserDetailPresenter.java

```diff
@@ -0,0 +1,36 @@
+package co.touchlab.droidconandroid.shared.presenter;
+
+import com.google.j2objc.annotations.Weak;
+
+import co.touchlab.droidconandroid.shared.tasks.FindUserTask;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.schedulers.Schedulers;
+
+public class UserDetailPresenter {
+
+    @Weak
+    private UserDetailHost host;
+    private FindUserTask task;
+    private CompositeDisposable disposables = new CompositeDisposable();
+
+    public UserDetailPresenter(UserDetailHost host, FindUserTask task) {
+        this.host = host;
```

Comment:
Maybe I shouldn't make the host a constructor argument for eventual Dagger's sake, and instead move it to the findUser method? What do you think @samhill303?